src/components/Submarine/Submarine.less

```diff
@@ -146,14 +149,14 @@
 		}
 
 		& > .lucid-Submarine-Primary {
-			flex-basis: 0px;
+			flex-basis: 0;
```

Comment:
flex-basis was set to `0px` to fix an IE bug where it doesn't treat `0` and `0px` the same. So we might need to ignore the lint rules for this line.